README.md

```diff
@@ -27,3 +27,20 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+## Aijun Hall
+
+An invariant for my recursive implementation of `fib()` could be that every single element `n` in
+the computed array `prev_fib_array` should be larger than the `n` element before it AFTER index 2. In other words, all elements in `prev_fib_array` should always be sorted in order from least at index 0, to largest at index -1.
```

Comment:
But how does this help you to show that the Fibonacci numbers are computed correctly?